frontend/src/assets/css/variables.css

```diff
@@ -0,0 +1,5 @@
+:root {
+  --text-light: #90a3bf;
+  --hover: #0084dd;
```

Comment:
![image](https://github.com/BinaryStudioAcademy/bsa-2023-bsa-talents/assets/78347072/c72b3bcc-2035-415c-a472-ebcb4d795bef)
Looks like for focused/hover state we have different collor